### DISCUSSION TITLE
selfhosted healthchecks and gunicorn

### DISCUSSION THREAD

**Discussion by pmatuszy** [healthchecks/healthchecks#616] [Q&A] [answered]
Hi
I just installed and configured healthchecks but would like to run it via gunicorn. I searched the internet and as I am database admin not web dev I have a problem how to run it.

Could you kindly show me how to run healthchecks using gunicorn (setup is done) - I just need a command to start the webserver...
Regards
Paul

**Comment by cuu508** [comment, marked as answer]
Hi @pmatuszy,

From the project root folder (where README is located), try

    gunicorn hc.wsgi:application

**Reply by pmatuszy** [reply]
thank you for this. This works!
Paul

**Comment by iammoizazam** [comment]
Hi , 

I'm trying to deploy a self-hosted version of health checks, I've tested the app in development mode, and it's working fine great.
Thumbs UP.   But I'm encountering an error when I change debug value to False it gives me "Server Error 500". 
I added my domain name to the allowed hostname list. moreover when I run manage check --deploy it shows me no error. 
please guide me through the solution.
thanks in advance.
